utils/mock-data-loader/gef/mandatoryCriteriaVersioned.js

```diff
@@ -1,9 +1,9 @@
 const MANDATORY_CRITERIA_VERSIONED = [
   {
-    version: 33,
-    createdAt: new Date('2021-01-03T00:00'),
+    version: 3,
+    createdAt: new Date('2024-11-12T00:00'),
     updatedAt: null,
-    isInDraft: false,
+    isInDraft: true,
```

Comment:
What's the reason for `isInDraft` being true here? Sorry if it's a silly question - been a while since i've worked on/see the mandatory criteria

Reply:
Actually I am going to revert it back to `false` since this will be fine or local and non-prod.